app/components/PlatformAppWebview.vue

```diff
@@ -11,7 +11,11 @@
     :partition="appPartition"
     preload="bundles/guest-api" />
   <div v-else class="platform-app-popped-out">
-    This app is currently popped out into an external window.
+    {{ $t("This app is currently popped out into an external window.") }}"
```

Comment:
Is the quotation mark at the end of this line supposed to be there?  I don't see it matched with anything.